src/components/TitleBar.tsx

```diff
@@ -40,14 +43,22 @@ const TitleBar: React.FC<TitleBarProps> = ({
       className={`h-titlebar  flex justify-between`}
       style={{ backgroundColor: "#303030" }}
     >
+
       <div
-        className=" flex"
+        className="flex"
         style={
           platform === "darwin"
             ? { marginLeft: "70px" }
```

Comment:
```suggestion
            ? { marginLeft: "60px" }
```

Reply:
Small suggestion to move the nav buttons closer to the system control buttons :)